startup/gui/menus.py

```diff
@@ -503,7 +503,6 @@ def __shaderNodeCreator( nodeName, shaderName ) :
 nodeMenu.append( "/Utility/Context Variables", Gaffer.ContextVariables, searchText = "ContextVariables" )
 nodeMenu.append( "/Utility/Delete Context Variables", Gaffer.DeleteContextVariables, searchText = "DeleteContextVariables" )
 nodeMenu.append( "/Utility/Time Warp", Gaffer.TimeWarp, searchText = "TimeWarp" )
-nodeMenu.append( "/Utility/Loop", Gaffer.Loop )
```

Comment:
I'm in favour of this change given some of the abominable uses of it we're finding. But I think we need to include a note in the change log to justify it. Perhaps something like this :


>Improvements
>------------------
>
>- NodeMenu : Removed Loop node. This node can have severe consequences for performance if used inappropriately.  Depending on the use case, the Collect nodes and others often provide a more performant alternative. The Loop node can still be created via the scripting API, but we recommend you consider the alternatives and/or request advice before using it.

Maybe this should also be limited to 0.59?

Reply:
Good idea on both counts. Updated.